vpr/src/route/rr_graph_storage.h

```diff
@@ -48,7 +48,7 @@
  *       This field is valid only for IPINs and OPINs and should be ignored  *
  *       otherwise.                                                          */
 struct alignas(16) t_rr_node_data {
-    int8_t cost_index_ = -1;
+    int16_t cost_index_ = -1;
```

Comment:
This should be put in another PR, to examine changing this independently of the other changes.